app/components/session/login_options_component.jsx

```diff
@@ -0,0 +1,41 @@
+import React from "react"
+import { Text, View } from "react-native";
+import { layoutStyles, loginOptionsStyles, styles } from "../../styles/styles"
+
+const LoginOptionsComponent = () => {
+  return(
+    <View style={[layoutStyles.backgroundLayout,loginOptionsStyles.loginLayoutCentering]}>
+      <View style={loginOptionsStyles.contentContainer}>
+        <View style={loginOptionsStyles.circleElementTitleContainer}> 
+          <View style={loginOptionsStyles.circleElement}>
+
+          </View>
+
+          <View style={loginOptionsStyles.Title}>
+              BGB
```

Comment:
When trying to run this on a phone, I get the error saying that these need to be wrapped in a `<Text></Text>` component. I don't think React Native lets us have free floating text.

Reply:
This applies to all the 'floating' text.